tests/baselines/reference/exportDefaultParenthesize.types

```diff
@@ -144,7 +144,7 @@ export default {
 
 === tests/cases/compiler/comma.ts ===
 export default {
->{    ['foo']: 42} : { ['foo']: number; }
+>{    ['foo']: 42} : { ["foo"]: number; }

```

Comment:
Actually, it seems to be using a completely different code path, because the output is `{ a: string } | { b: string }`. I *guess* this is OK, but I didn't expect it. See the latest commits for the declaration baseline.